back/app/services/permissions/permissions_fields_service.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+module Permissions
+  class PermissionsFieldsService
+    # To be called from rake task when enabling custom_permitted_by feature flag or on tenant creation
+    def change_permissions_to_custom
```

Comment:
Is this called every time the feature is enabled/disabled? Suppose we would have an unexpected failure somewhere in the middle of this process, would the consequences be bad for the platform (for example the GET projects endpoint no longer working properly)?

Do you think it could be possible to make updates only when one permissions' permitted_by is changed to "custom"? Then, potential issues would only affect the corresponding permitable. I'm not convinced of this approach either though.

Reply:
Could it be possible to persist all supported default fields and only filter those related to the permitted_by?

Reply:
Today, we have so much inconsistent data, it might happen that `save!` will fail at some point.